src/main/java/com/onarandombox/MultiverseCore/utils/permission/PermissionsRegistrar.java

```diff
@@ -21,15 +21,15 @@ public class PermissionsRegistrar {
 
     public static void setup() {
         worldPermissions = new ArrayList<>() {{
-            worldAccessPermission = new PrefixPermission("multiverse.access.", "Allows access to a world.");
-            worldGamemodeBypassPermission = new PrefixPermission("mv.bypass.gamemode.", "Allows bypassing of gamemode restrictions.", PermissionDefault.FALSE);
-            worldExemptPermission = new PrefixPermission("multiverse.exempt.", "A player who has this does not pay to enter this world.");
-            worldPlayerLimitBypassPermission = new PrefixPermission("mv.bypass.playerlimit.", "Allows bypassing of player limit restrictions.");
+            add(worldAccessPermission = new PrefixPermission("multiverse.access.", "Allows access to a world."));
+            add(worldGamemodeBypassPermission = new PrefixPermission("mv.bypass.gamemode.", "Allows bypassing of gamemode restrictions.", PermissionDefault.FALSE));
+            add(worldExemptPermission = new PrefixPermission("multiverse.exempt.", "A player who has this does not pay to enter this world."));
+            add(worldPlayerLimitBypassPermission = new PrefixPermission("mv.bypass.playerlimit.", "Allows bypassing of player limit restrictions."));
         }};
```

Comment:
I think we should avoid double brace initialization as it creates a subclass of ArrayList with an instance initializer which is generally overkill for just about any use case. Also, assignment inside a method call should be avoided. Each assignment should always be on its own line.